db/init_db.js

```diff
@@ -1,25 +1,74 @@
 // code to build and initialize DB goes here
 const {
-  client
+  client,
   // other db methods 
+  createAlbums,
+  createGenres,
+  createAlbumGenres,
+  createUser
 } = require('./index');
 
 async function buildTables() {
   try {
-    client.connect();
-
-    // drop tables in correct order
+    console.log("Building tables...");
+    await client.connect();
 
     // build tables in correct order
 
+    await client.query(`
+    DROP TABLE IF EXISTS users;
+    DROP TABLE IF EXISTS genre_albums;
+    DROP TABLE IF EXISTS genres;
+    DROP TABLE IF EXISTS products;
+
+    CREATE TABLE products (
```

Comment:
I would prefer albums instead of products. or change genres_albums for genres_products.